packages/react/src/components/footer.tsx

```diff
@@ -0,0 +1,279 @@
+  FooterAgencyMediaLinksLogoPathProps
+> = ({ icons, className }) => {
+  return (
+    <div className={clx("text-txt-black-700 flex gap-3", className)}>
+      {icons.map((item, index) => (
+        <a
+          key={index}
+          href={item.href}
+          target="_blank"
+          rel="noopener noreferrer"
+          className=""
+        >
+          {item.icon}
+        </a>
+      ))}
+    </div>
+  );
+};
+
+const FooterDisclaimer: FunctionComponent<FooterDisclaimer> = ({
+  children,
+  className,
+}) => (
+  <div
+    className={clx(
+      "text-txt-black-500 max-sm:px-4.5 flex flex-col justify-between gap-6 pt-8 text-sm lg:flex-row",
+      className,
+    )}
+  >
+    {children}
+  </div>
+);
+
+const FooterDisclaimerGroup: FunctionComponent<FooterDisclaimerGroup> = ({
+  children,
+  className,
+}) => (
+  <div
+    className={clx(
+      "flex flex-col gap-2 lg:flex-row lg:items-center",
+      className,
+    )}
+  >
+    {children}
+  </div>
+);
+
+const FooterDisclaimerGroupRights: FunctionComponent<
+  FooterDisclaimerGroupRights
+> = ({ children, className }) => (
+  <>
+    <p className={className}>{children}</p>
+    <span className="bg-otl-divider hidden h-3 w-px lg:block"></span>
+  </>
+);
+
+const FooterDisclaimerGroupDisclaimer: FunctionComponent<
+  FooterDisclaimerGroupDisclaimer
+> = ({ children, className }) => (
+  <div
+    className={clx(
+      "text-txt-black-700 flex flex-wrap gap-x-3 gap-y-2",
+      className,
+    )}
+  >
+    {children}
+  </div>
+);
+
+const FooterDisclaimerLastUpdate: FunctionComponent<
+  FooterDisclaimerLastUpdate
+> = ({ children, className }) => (
+  <time className={className} dateTime={new Date().toISOString()}>
+    {children}
+  </time>
+);
+
+const FooterGeneralLink: FunctionComponent<FooterGeneralLink> = ({
+  children,
+  href,
+  className,
+}) => (
+  <a
+    href={href}
+    className={clx(
+      "text-txt-black-700 hover:text-txt-black-900 text-sm hover:underline hover:underline-offset-auto",
+      className,
+    )}
+    target="_blank"
+    rel="noopener noreferrer"
+  >
+    {children}
+  </a>
+);
+
+const Footer: FunctionComponent<Footer> = ({ children, className }) => (
+  <div
+    className={clx(
+      "border-otl-gray-200 bg-bg-gray-50 border-t print:hidden",
+      className,
+    )}
+  >
+    <div className="divide-otl-gray-200 bg-bg-gray-50 container mx-auto divide-y px-6 py-8 max-sm:px-0 lg:px-[60px] lg:py-16">
+      {children}
+    </div>
+  </div>
+);
+
+export {
+  Footer,
+  FooterContent,
+  FooterContentAgencySection,
+  FooterAgencyHeader,
+  FooterAgencyAddress,
+  FooterAgencyMediaLinks,
+  FooterAgencyMediaLinksLogoPath,
+  FooterContentLinkSection,
+  FooterContentLinkSectionWrapper,
+  FooterContentLinkSectionTitle,
+  FooterContentLinkSectionContent,
+  FooterGeneralLink,
+  FooterDisclaimer,
+  FooterDisclaimerGroup,
+  FooterDisclaimerGroupRights,
+  FooterDisclaimerGroupDisclaimer,
+  FooterDisclaimerLastUpdate,
+};
```

Comment:
Too many exports. I think certain items can be taken from other component OR does not require it's own component. 

Eg. 
1. `FooterAgencyMediaLink` => `Link`
2. `FooterGeneralLink` => `Link`
3. `FooterDisclaimerGroupRights` 
4. `FooterDisclaimerGroupDIsclaimer` 
5. `FooterDisclaimerLastUpdate` 
6. etc

Reply:
ok boss, i try reducing it.